rm/ut/rm_service.c

```diff
@@ -73,6 +70,9 @@ static void rm_service_stop(struct m0_rpc_server_ctx *sctx)
 
 static void rm_svc_server(const int tid)
 {
+
+	struct m0_net_xprt *xprt = m0_net_xprt_default_get();
+	sctx.rsx_xprts = &xprt; 
```

Comment:
"xprt" is an automatic variable, it goes out of scope when rm_svc_server() returns. Are you sure this is safe?

Reply:
This is a bug.

But `sctx` happened to be initialized, and finalized only this in this function. So, the test code did not crash.
But yes, this should be fixed.

Reply:
incorporated